k8s/workloads/moderator/moderator-ingress.yaml

```diff
@@ -34,15 +34,10 @@ spec:
       scope:
         enabled: true
       config:
-        use-proxy-protocol: "false"
+        compute-full-forwarded-for: "true"
+        enable-real-ip: "true"
```

Comment:
will be curious if this works as you expect, if so I'll probably flow to other services as well.  🤞